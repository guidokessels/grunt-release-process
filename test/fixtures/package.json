{
  "name": "grunt-release-process",
  "version": "0.0.41"
}
