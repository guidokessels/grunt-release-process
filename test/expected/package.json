{
  "name": "grunt-release-process",
  "version": "0.0.42"
}
